simulation/traffic_simulator/src/entity/entity_manager.cpp

```diff
@@ -253,14 +253,24 @@ auto EntityManager::getLongitudinalDistance(
   bool include_adjacent_lanelet, bool include_opposite_direction) -> std::optional<double>
 {
   if (!include_adjacent_lanelet) {
-    auto forward_distance = hdmap_utils_ptr_->getLongitudinalDistance(
-      static_cast<LaneletPose>(from), static_cast<LaneletPose>(to));
+    auto to_canonicalized = static_cast<LaneletPose>(to);
```

Comment:
```suggestion
    const auto to_canonicalized = static_cast<LaneletPose>(to);
```

Reply:
I'm not able to add `const` to the `to_canonicalized` variable because if `to.hasAlternativeLaneletPose()` then assign to `to_canonicalized` alternative lanelet pose